msteams-platform/resources/teams-updates.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Microsoft Teams updates
+description: Learn about the latest updates to Microsoft Teams.
+author: v-ypalikila
+ms.author: lajanuar
+ms.localizationpriority: medium
+ms.topic: reference
+---
+# Microsoft Teams update
+
+[The new Microsoft Teams client](https://www.microsoft.com/en-us/microsoft-365/blog/2023/03/27/welcome-to-the-new-era-of-microsoft-teams/) is reimagined from the ground up with performance in mind. It's faster, simpler, smarter, and flexible to provide better experience for your apps and users. The new Teams client supports all the existing Teams app capabilities except Adaptive Card tabs. If you have an app that runs inside the Classic Teams, the app will most likely run in the new Teams client without any issues.
```

Comment:
Please add a link for "Adaptive Card Tabs" to make it clear what capability we're talking about here:
[Adaptive Card tabs](/microsoftteams/platform/tabs/how-to/build-adaptive-card-tabs)

Reply:
Updated the link to [Adaptive Card tabs](https://learn.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/build-adaptive-card-tabs) in the note:

![image](https://github.com/MicrosoftDocs/msteams-docs/assets/93579821/c0104280-1f8a-4ca0-bcc6-e7b94125dd01)